app/models/grda_warehouse/tasks/push_clients_to_cas.rb

```diff
@@ -341,6 +342,7 @@ def value_display_for(key, value)
           :date_of_birth,
           :dob_quality_code,
           :alternate_names,
+          :encampment_decomissioned,
```

Comment:
Hides it from the read-only attribute table on the CAS readiness page. If we start ingesting it from somewhere else, we'll need to remove this line to make it show up on the table.